Tests.NetCore/TextSerializerTests.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Prometheus.Tests;
+
+[TestClass]
+public class TextSerializerTests
+{
+    [TestMethod]
+    public async Task ValidateTextFmtSummaryExposition_HappyPath()
+    {
+        var result = await TestCase.Run(factory =>
+        {
+            var summary = factory.CreateSummary("boom_bam", "", new SummaryConfiguration
+            {
+                LabelNames = new[] { "blah" },
+                Objectives = new[]
+                {
+                    new QuantileEpsilonPair(0.5, 0.05),
+                }
+            });
+
+            summary.WithLabels("foo").Observe(3);
+        });
+        // TODO help has a trailing whitespace before the newline
```

Comment:
What are your thoughts on this? Planned for followup in next iterations, ignore for now? I would prefer to see some fix for it already. It is only help text but historically there have been apps taking a dependency on help text details and freaking out when it is unusual.